### PR TITLE
added new notebook and merge of trainer

### PR DESCRIPTION
**Pull request by philschmid**
# What does this PR do?

* Adds a new Notebook for SageMaker
* Adjusts documentation for the latest merge of `SageMakerTrainer` and `Trainer`

### CODE REVIEW

File: docs/source/sagemaker.md [hunk 1]
```diff
@@ -193,7 +193,9 @@ You can find here a list of the official notebooks provided by Hugging Face.
...
-| [Distributed Training Data Parallelism Tensorflow](https://github.com/huggingface/notebooks/blob/master/sagemaker/07_tensorflow_distributed_training_data_parallelism/sagemaker-notebook.ipynb) | End-to-End distributed binary Text-Classification example using `Keras` and `TensorFlow`                         |
+| [Distributed Training Data Parallelism Tensorflow](https://github.com/huggingface/notebooks/blob/master/sagemaker/07_tensorflow_distributed_training_data_parallelism/sagemaker-notebook.ipynb) | End-to-End distributed binary Text-Classification example using `Keras` and `TensorFlow`                    
+| [Distributed Seq2Seq Training with Data Parallelism and BART](https://github.com/huggingface/notebooks/blob/master/sagemaker/08_distributed_summarization_bart_t5/sagemaker-notebook.ipynb) | End-to-End distributed summarization example `BART-large` and 🤗 Transformers example script for `summarization`                        |
```
Comment (LysandreJik) [inline docs/source/sagemaker.md:197]: Should the title mention that it's on Sagemaker?

File: docs/source/sagemaker.md [hunk 2]
```diff
@@ -230,16 +232,12 @@ huggingface_estimator = HuggingFace(
 
 ### Distributed Training: Model-Parallel
 
-You can use [SageMaker Model Parallelism Library](https://aws.amazon.com/blogs/aws/amazon-sagemaker-simplifies-training-deep-learning-models-with-billions-of-parameters/) out of the box for distributed training. We extended the Trainer API to the [SageMakerTrainer](https://github.com/huggingface/transformers/blob/461e8cacf94d1f76367cc9ba2cfd5b9bd3641c81/src/transformers/sagemaker/trainer_sm.py#L72) to use the model parallelism library. Therefore you only have to change the imports in your `train.py`.
+You can use [SageMaker Model Parallelism Library](https://aws.amazon.com/blogs/aws/amazon-sagemaker-simplifies-training-deep-learning-models-with-billions-of-parameters/) out of the box for distributed training.We added the functionality of Model Parallelism directly into the [Trainer](https://huggingface.co/transformers/main_classes/trainer.html). If your `train.py` uses the [Trainer](https://huggingface.co/transformers/main_classes/trainer.html) API you only need to define the distribution parameter in the HuggingFace Estimator.
```
Comment (LysandreJik) [inline docs/source/sagemaker.md:235]:
```suggestion
You can use [SageMaker Model Parallelism Library](https://aws.amazon.com/blogs/aws/amazon-sagemaker-simplifies-training-deep-learning-models-with-billions-of-parameters/) out of the box for distributed training. We added the functionality of Model Parallelism directly into the [Trainer](https://huggingface.co/transformers/main_classes/trainer.html). If your `train.py` uses the [Trainer](https://huggingface.co/transformers/main_classes/trainer.html) API you only need to define the distribution parameter in the HuggingFace Estimator.  
```

**Review by LysandreJik — APPROVED** [review]
Perfect, thanks for updating! LGTM!

File: docs/source/sagemaker.md (hunk 1 above)
Comment (philschmid) [reply docs/source/sagemaker.md:197]: I don´t think so since it is on the sagemaker docs page and in the notebooks repo under `sagemaker/`

**Pull request merged**